networkx/algorithms/traversal/breadth_first_search.py

```diff
@@ -37,8 +37,8 @@ def generic_bfs_edges(G, source, neighbors=None, depth_limit=None, sort_neighbor
         that returns an iterator over some or all of the neighbors of a
         given node, in any order.
 
-    depth_limit : int, optional(default=len(G))
-        Specify the maximum search depth
+    depth_limit : int
+        Specify the maximum search depth, by default it searches the all graph.
```

Comment:
I'm unsure this should change. The change in clarity is minor and other doc strings are using the original format. [See the functions in this file](https://github.com/networkx/networkx/pull/6359/files#diff-1102936e5b1885b644337511d00fbf83f9bf32d111b00b261c1136a6f4db1d69R112). Seems to be consistent across other docstrings in nx I checked too.